evaluation/swe_bench/run_infer.py

```diff
@@ -90,7 +90,7 @@ def get_config(
     instance: pd.Series,
     metadata: EvalMetadata,
 ) -> AppConfig:
-    SWE_BENCH_CONTAINER_IMAGE = 'ghcr.io/openhands/eval-swe-bench:full-v1.2.1'
+    SWE_BENCH_CONTAINER_IMAGE = 'ghcr.io/all-hands-ai/eval-swe-bench:full-v1.2.1'
```

Comment:
Probably need to revert this one

```suggestion
    SWE_BENCH_CONTAINER_IMAGE = 'ghcr.io/opendevin/eval-swe-bench:full-v1.2.1'
```

Reply:
Hmm why would opendevin still be used? 🤔

Reply:
Because this was the old container and we didn't move the container here